packages/react-data-grid/src/ColumnComparer.js

```diff
@@ -1,11 +1,11 @@
-import { isValidElement } from 'react';
+import { isElement } from 'react-is';
 
 export function sameColumn(a, b) {
   let k;
 
   for (k in a) {
     if (a.hasOwnProperty(k)) {
-      if ((typeof a[k] === 'function' && typeof b[k] === 'function') || (isValidElement(a[k]) && isValidElement(b[k]))) {
+      if ((typeof a[k] === 'function' && typeof b[k] === 'function') || (isElement(a[k]) && isElement(b[k]))) {
```

Comment:
Shouldn't these typeof checks be replaced with `isValidElementType`?

Reply:
That makes sense. For this PR I just replaced `React.isValidElement` with `ReactIs.isElement` to be on  the safer side. I will revisit this once we clean up column metrics logic.